src/PABLO/tree_constants.cpp

```diff
@@ -56,8 +56,8 @@ TreeConstants::instance(uint8_t dim) {
 const TreeConstants::Instances &
 TreeConstants::instances() {
 	static TreeConstants::Instances instances = {{
-		TreeConstants(0),
-		TreeConstants(0),
+		TreeConstants(2),
+		TreeConstants(2),
```

Comment:
@andrea-iob these issues were picked up from running the clang memory sanitiser. It may be worth integrating a memory sanitiser in your pipeline to pick these undefined behaviours up so we can remove them?